services/packagist/packagist-dependency-version.service.js

```diff
@@ -0,0 +1,204 @@
+import Joi from 'joi'
+import { optionalUrl } from '../validators.js'
+import { NotFound } from '../index.js'
+import {
+  allVersionsSchema,
+  BasePackagistService,
+  customServerDocumentationFragment,
+} from './packagist-base.js'
+
+const queryParamSchema = Joi.object({
+  server: optionalUrl,
+  version: Joi.string(),
+}).required()
+
+export default class PackagistDependencyVersion extends BasePackagistService {
+  static category = 'platform-support'
+
+  static route = {
+    base: 'packagist/dependency-v',
+    pattern: ':user/:repo/:dependency+',
+    queryParamSchema,
+  }
+
+  static examples = [
+    {
+      title: 'Packagist Dependency Version Support',
```

Comment:
I know the titles for the old PHP version badge included the word "support", but looking over the titles for everything else in the "Platform & Version Support" category, I think we should drop the word "support" on all 4 titles.